src/main/views/subscription-confirmed.njk

```diff
@@ -0,0 +1,37 @@
+{% extends "template.njk" %}
```

Comment:
This page looks a bit different from the one in the JIRA ticket. Just want to double check the below is correct (titles, bullet points)?

Reply:
I have taken the screen from the wireframe on herokuapp

Reply:
heroku should be used as a lose guide, the standard we should be using is the AC on the ticket and the screenshots provided in the ticket, Heroku is used to prototype different designs, the approved and needing developing designs are the ones provided on the ticket, 9 times out of 10 its the ticket we should be using unless stated otherwise

Reply:
spoke to Nigel. He will add comments in the ticket